clang/test/CodeGen/ptrauth-intrinsic-sign-constant.c

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -triple arm64-apple-ios -fptrauth-intrinsics -emit-llvm %s  -o - | FileCheck %s
```

Comment:
It would be nice to also add a run line against `aarch64` or `aarch64-elf` generic triple in this and other tests. The logic tested is not apple-specific and locally I've ensured that adding this run line does not break tests.

Reply:
@kovdan01 If you're having a RUN line locally, can you just suggest a change here?

Reply:
> @kovdan01 If you're having a RUN line locally, can you just suggest a change here?

Updated the comment, thanks